ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-composite-answer/activityCompositeAnswer.component.html

```diff
@@ -1,18 +1,24 @@
-<ddp-question-prompt [block]="block"></ddp-question-prompt>
+<ddp-question-prompt *ngIf="!isGridLayout()" [block]="block"></ddp-question-prompt>
 <ng-container *ngFor="let childBlockRow of childQuestionBlocks; let row = index">
     <p *ngIf="row > 0" class="Title-text PageContent-subtitle Normal">
         <span>{{ block.additionalItemText }}</span>
         <button *ngIf="!readonly" mat-icon-button (click)="removeRow(row)">
             <mat-icon class="ddp-close-button">close</mat-icon>
         </button>
     </p>
-    <div [ngClass]="['ddp-answer-container', setOrientationClass(block.childOrientation)]">
+    <div class="ddp-answer-container"
+         [ngClass]="setOrientationClass(block.childOrientation)"
+         [gdColumns]="gridLayoutColumnConfig()">
         <ng-container *ngFor="let childBlock of childBlockRow; let column = index">
+            <div *ngIf="block.tabularSeparator && column === 1" class="tabular-separator">
+                <span>{{block.tabularSeparator}}</span>
+            </div>
```

Comment:
Don't you think that the separator can be shown more than once ? 
Like 
`"Year" or "Age" or "Something else"` 

In this cases, to be more general, we can use `odd` definition (instead of `column === 1`):
```
<ng-container *ngFor="let childBlock of childBlockRow; let column = index; let odd=odd;">
            <div *ngIf="block.tabularSeparator && odd" class="tabular-separator">
                <span>{{block.tabularSeparator}}</span>
            </div>
```

Reply:
Yeah. I that situation did cross my mind, but it does not appear that we need that in FON. And I also remembered this particular case that we need to support:
![Screen Shot 2022-05-26 at 11 52 46 AM](https://user-images.githubusercontent.com/29984380/170525803-1950a46f-26b8-489b-be9a-834a262f6027.png)
So that checkbox is could be part of the composite, so 2nd "Or" would not make sense.
I mean, when I was thinking about this particular problem, I had thought that this is such a weird case that we should just deal with it with custom CSS that inserts the "Or" in there.
You decided to handle it in the API and code as actual HTML element which once I saw it in your code I thought it was better idea.
Regardless, if we need multiple separators. Or if the separator has to go between, say, the 2nd question and the 3rd question, we can revisit the API and add more info about it. I am inclined to just go with approach that we have.